### PR TITLE
wallet: main screen live data binding

### PR DESCRIPTION
**Pull request by oskarth**
wallet: main screen live data binding

Initial utility for live data binding in the new wallet. See https://github.com/status-im/status-react/issues/1612

- Add prices namespace to get fiat prices from Cryptocompare
- Events to init wallet balance and load prices
- Listen to these events in wallet main view
- Show accurate ETH balance, USD value and %change from yesterday

status: ready

Testing / expected behavior:
- If you have ETH in account, it should show up in wallet
- USD value should be accurate
- %change should roughly correspond to % ETH movement last 24h

### CODE REVIEW

**Comment by oskarth** [comment]
@jeluard fixed the things we talked about yesterday

Ready to be merged as far as I'm concerned, in order to share wallet progress.

File: src/status_im/ui/screens/wallet/main/views.cljs [hunk 1]
```diff
@@ -72,20 +73,46 @@
...
      [rn/text {:style st/asset-section-title} "Assets"]
      [rn/list-view {:dataSource      (lw/to-datasource assets)
                     :renderSeparator (when platform/ios? (render-separator-fn (count assets)))
                     :renderRow       render-row-fn}]]))
 
+(defn wallet->eth-balance [wallet]
+  (when-let [balance (:balance wallet)]
+    (money/wei->ether balance)))
+
+(defn wallet+prices->portfolio-value [{:keys [balance]} {:keys [price]}]
```
Comment (rasom) [inline src/status_im/ui/screens/wallet/main/views.cljs]: Let's call it `portfolio-value`. `wallet` and `prices` are parameters, when we call it it looks weird `(wallet+prices->portfolio-value wallet prices)`

Comment (rasom) [reply src/status_im/ui/screens/wallet/main/views.cljs]: The same for `wallet->eth-balance` and `prices->portfolio-change`

Comment (oskarth) [reply src/status_im/ui/screens/wallet/main/views.cljs]: Absolutely, will fix

**Review by rasom — APPROVED** [review]

**Review by jeluard — APPROVED** [review]

**Comment by oskarth** [comment]
@rasom fixed things you mentioned and rebased. As I rebased there was a regression with `accounts/current-account-id` causing `invalid address` error. I mitigated it by adding a safeguard in `get-balance` function and namespacing destructuring.

**Comment by annadanchenko** [comment]
ok on iOS (iphone 6s) and Android (Google Pixel XL)
checked for 0 ETH, 1 ETH. 
- USD value and total value change corresponds with values on https://www.cryptocompare.com/coins/eth/overview/USD for the moment when app was launched. 
- Value for ETH in Assets corresponds with ETH available in the wallet
- No refresh is there yet, to see any updates need to re-launch app

**Pull request merged**